.github/CHANGELOG.md

```diff
@@ -130,10 +205,10 @@
 
```

Comment:
^---- this is inaccurate, since we haven't changed how QNode queuing works yet

Reply:
Will remove! Good catch